tools/scripts/bootstrap-lib.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
We're trying to move away from bash scripts and to use [zx](`https://github.com/google/zx`) instead.